src/styles/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-items: center;
+  width: auto;
+  justify-content: space-between;
+  line-height: 16px;
+  padding: 16px 20px 0 16px;
+
+  &__star {
+    background-size: cover;
+    width: 12px;
+    height: 12px;
+    margin-left: 4px;
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__star--inactive {
+    background-image: url(../images/star.svg);
+  }
+
+  &__reviews {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    display: flex;
+    margin-left: 16px;
+    color: $product-font-color;
+    height: 16px;
+    padding-top: 2px;
```

Comment:
instead of `padding` you should use `flex` for a parent element and align as you need
```suggestion
```